charts/brigade-github-app/values.yaml

```diff
@@ -3,7 +3,7 @@ rbac:
   enabled: false
 
 # Image configuration
-registry: technosophos
+registry: deis
```

Comment:
👍 -- there may be another instance, I think in docs/README.md still referencing `technosophos`

Reply:
Fixed